wowchemy/layouts/partials/site_footer.html

```diff
@@ -1,4 +1,46 @@
 <footer class="site-footer">
+  {{/* Look for social icons that should be displayed in the footer.
+       For now just look for a superuser. If more than one superuser
+       is defined, run through all of them. */}}
+
+  {{ $displayicons := false }}
+  {{ range where (where site.Pages "Section" "authors") ".Params.superuser" true }}
```

Comment:
You've proposed running the exact same loop twice, on line 7 and then again on line 17. Can you rewrite this more efficiently, following engineering best practices?

Reply:
I must have been tired! Fixed